server/acl_casbin_policy_dba.csv

```diff
@@ -80,6 +80,7 @@ p, DBA, /vcs/{id}, GET
 p, DBA, /vcs/{id}, PATCH
```

Comment:
We also need to add the route for developer. Since developer can be a project owner and can setup VCS integration. Thus it needs to fetch the external repository list.